_community_members/upasagar.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Sagar Upadhyaya
+short_name: upasagar
+photo: '/assets/media/community/members/upasagar.jpg'
+title: 'OpenSearch Community Member: Sagar Upadhyaya
+primary_title: Sagar Upadhyaya
+breadcrumbs:
+  icon: community
+  items:
+    - title: Community
+      url: /community/index.html
+    - title: Members
+      url: /community/members/index.html
+    - title: 'Sagar Upadhyaya&apos;s Profile'
+      url: '/community/members/sagar-upadhyaya.html'
+github: uselmh
+job_title_and_company: 'Social media specialist for the OpenSearch Project'
```

Comment:
@sgup432 wow social media star. Are these correct?

Reply:
@dzane17 Do you have any doubt? 😁